agents/gce/fence_gce.py

```diff
@@ -97,6 +206,25 @@ def main():
 
 	run_delay(options)
 
+	# Prepare logging
+	if options.get('--stackdriver-logging') is not None:
+		try:
+			import google.cloud.logging.handlers
+			client = google.cloud.logging.Client()
+			handler = google.cloud.logging.handlers.CloudLoggingHandler(client, name=hostname)
+			handler.setLevel(logging.INFO)
+			formatter = logging.Formatter('gcp:stonish "%(message)s"')
+			handler.setFormatter(formatter)
+			root_logger = logging.getLogger()
+			if options.get('--verbose') is None:
+				root_logger.setLevel(logging.INFO)
+				logging.getLogger("googleapiclient").setLevel(logging.ERROR)
```

Comment:
This should be put outside stackdriver-logging, as we dont want to see these warnings for every run.

Reply:
ok, I'll update the PR.